src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,37 @@
+<div class="footer">
+  <div class="contact">
+    <div class="footer-img">
+      <img src="../../../img/footer.png" />
+    </div>
+    <div class="message">
+      <div class="message-inner">
+        <p class="small">
+          〒532-0011　大阪市淀川区西中島６丁目8-23テスト <br />TEL.06-1234-5678
+          FAX.06-1234-5678　営業時間/9:00 - 22:00　定休日/不定休
+        </p>
+      </div>
+    </div>
+    <div class="icon-list">
+      <i class="fab fa-facebook-f"></i>
```

Comment:
```suggestion
       <a href="#" ><i class="fab fa-facebook-f"></i></a>
```
機能的にはリンクなので、このようにするのが適切です。
今回はデモなので一旦このままでOKです。